specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/examples/ListUsagesByVMFamilyResult.json

```diff
@@ -0,0 +1,298 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "location": "eastus",
+    "api-version": "2018-11-19"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "unit": "Count",
+            "currentValue": 2,
+            "limit": 100,
+            "name": {
+              "value": "Clusters",
+              "localizedValue": "Clusters"
+            },
+            "resourceType": "TotalClusters",
```

Comment:
According to the swagger, `resourceType` and `resourceGroupName` are supposed to be under a `properties` key like this.
```json
"properties": {
  "resourceType": "TotalClusters",
  "resourceGroupName": "rg"
}
```
This is causing model validation failures, see the [log](https://dev.azure.com/azure-sdk/public/_build/results?buildId=30974).
Comment applies to all items in the array.